client/src/components/PublicLobbyList.tsx

```diff
@@ -50,6 +50,7 @@ export function PublicLobbyList(props: PublicLobbyListProps) {
           </tr>
           {lobbies
             .filter((l) => readyRooms.has(l.roomId))
+            .sort((a, b) => (new Date(b.createdAt).getTime() || 0) - (new Date(a.createdAt).getTime() || 0))
```

Comment:
we want to filter ended games also right?

Reply:
Now that I added destroy room, ended games will disappear 15s after the game ends. Want to leave this un filtered so users can see that GAME ENDED state get updated and shown briefly (in case that's a feature they want in their lobby lists)